docs/src/main/tut/datatypes/repl.md

```diff
@@ -0,0 +1,367 @@
+---
+layout: docs
+title:  "Learning Algebird at the REPL"
+section: "data"
+---
+
+# Learning Algebird Monoids at the REPL
```

Comment:
this is the "REPL" page from the wiki, with a nice new index, converted to `tut`.